src/styles/blocks/container-of-cards.scss

```diff
@@ -0,0 +1,101 @@
+@import "../utils/variables";
+
+.container-of-cards {
+  display: flex;
+  width: $container-size;
+  margin: auto;
+  flex-wrap: wrap;
+  padding: 50px $container-pad;
+  
+  .container {
+    margin-right: $card-margin;
+    
+    .card {
+      display: flex;
+      width: $card-size;
+      flex-direction: column;
+      border-radius: 4px;
+      border: 1px solid #f3f3f3;
+      
+      &__icon {
+        width: 160px;
+        height: 134px;
+        margin-bottom: 40px;
+        align-self: center;
+      }
+    
+      &__theme {
+        font-weight: 500;
+        font-size: 12px;
+        line-height: 18px;
+        color: $black-color;
+        margin-bottom: 4px;
+      }
+    
+      &__prod-cod {
+        font-size: 10px;
+        color: #616070;
+        line-height: 14px;
+        margin-bottom: 16px;
+        font-weight: 400;
+      }
+    
+      &__price-inform {
+        display: flex;
+        justify-content: space-between;
+        margin-bottom: 16px;
+    
+        .card__price {
+          font-size: 12px;
+          line-height: 18px;
+          color: #616070;
+        }
+    
+        .card__amount {
+          font-size: 16px;
+          font-weight: 700;
+          line-height: 18px;
+          color: $black-color;
+        }
+          
+      }
+    
+      &__link {
+        display: flex;
+        width: 166px;
```

Comment:
do not hardcode width
if card will become wider - this hardcoded value remains the same
it should be 100% width

Reply:
ok , thanks!